### PR TITLE
Evolution - Wastewater surveillance

### PR DESCRIPTION
**Pull request by RLordan**
Hey @rando2 see edits. If you wanted a figure, although probably not necessary, there is one for the wastewater surveillance nationwide here: https://covid.cdc.gov/covid-data-tracker/#wastewater-surveillance

### CODE REVIEW

File: content/09.evolution.md [hunk 1]
```diff
@@ -386,7 +386,16 @@ The SARS-CoV-2 pandemic has presented many unprecedented scientific opportunitie
...
 The information obtained from genetic, bioinformatics, and evolutionary analysis has played a significant role in shaping the global pandemic response [@doi:10.1038/s41592-022-01444-z; @doi:10.1038/s41586-021-04188-6; @doi:10/hskp].
-<!-- Wastewater surveillance? Implications for preventing future pandemics? Vaccine response time?-->
+For example, wastewater surveillance has emerged as a potential epidemiological tool to monitor SARS-CoV-2 spread over large regions, complementing clinical surveillance [@doi:10.1016/j.scitotenv.2020.138875; @doi:10.1016/j.coesh.2020.09.006; @doi:10.1016/j.coesh.2020.09.003].
+Humans shed SARS-CoV-2 viral RNA in feces [@doi:10.1016/S2468-1253(20)30083-2], which is detectable in wastewater.
+Protocols have been developed to safely and reproducibly isolate and quantify SARS-CoV-2 in samples obtained from wastewater processing plants [@doi:10.1371/journal.pone.0257454; @doi:10.1016/j.coesh.2020.09.003].
```
Comment (RLordan) [inline content/09.evolution.md:391]:
```suggestion
Protocols have been developed to safely and reproducibly isolate and quantify SARS-CoV-2 in samples obtained from wastewater processing plants [@doi:10.1371/journal.pone.0257454; @doi:10.1016/j.coesh.2020.09.003].
```

**Comment by RLordan** [comment]
Super thanks @rando2

**Comment by RLordan** [comment]
The only thing I thought about after was, should I mention a specific variant that can be monitored this way... considering its an evolution paper - @rando2

File: content/09.evolution.md [hunk 2]
```diff
@@ -386,7 +386,16 @@ The SARS-CoV-2 pandemic has presented many unprecedented scientific opportunitie
...
+To date, studies show that wastewater surveillance is an effective tool to monitor SARS-CoV-2 spread over large sewersheds [@doi:10.1016/j.scitotenv.2020.138875; @doi:10.1016/j.coesh.2020.09.006; @doi:10.1016/j.coesh.2020.09.003; @doi:10.1039/D1EW00747E]. 
+Indeed, data from a study in New York City indicated that wastewater SARS-CoV-2 detection correlated with clinical detection of infection  [@doi:10.1039/D1EW00747E].
+Similar studies have been conducted in Nevada [@doi:10.1016/j.wroa.2020.100086] and Boston [@doi:10.1128/mSystems.00614-20].
+To date, studies have shown that factors such as temperature, the travel time of wastewater, and diurnal variability may affect detection of SARS-CoV-2 [@doi:10.1016/j.scitotenv.2020.138875; @doi:10.1016/j.wroa.2020.100086].
+Due to its demonstrated utility so far, the United States CDC established the National Wastewater Surveillance System (NWSS), which has emerged as an important surveillance tool for SARS-CoV-2 spread [@url:https://www.cdc.gov/healthywater/surveillance/wastewater-surveillance/wastewater-surveillance.html]. 
+
+Implications for preventing future pandemics? Vaccine response time?-->
```
Comment (agitter) [inline content/09.evolution.md:398]:
```suggestion
<!-- Implications for preventing future pandemics? Vaccine response time?-->
```
Making sure this part stays invisible as a comment

**Comment by rando2** [comment]
@rlo

**Comment by RLordan** [comment]
> Apparently I can't type today! @RLordan that might be interesting if it's used to track turnover of dominant variants? But otherwise, I think it's fine, since we hit genomic surveillance hard in the conclusion! (To me, genomics and evolution are closely tied together even if the tech can also be used for other purposes!)

Fair point, take a look at these two and see if it is worth a mention https://www.medrxiv.org/content/10.1101/2022.02.18.22271188v1
https://doi.org/10.1101/2021.12.21.21268077

File: content/09.evolution.md [hunk 3]
```diff
@@ -386,7 +386,16 @@ The SARS-CoV-2 pandemic has presented many unprecedented scientific opportunitie
...
+Humans shed SARS-CoV-2 viral RNA in feces [@doi:10/ggq8zp] that can be detected in wastewater.
+Protocols have been developed to safely and reproducibly isolate and quantify SARS-CoV-2 in samples obtained from wastewater processing plants [@doi:10.1371/journal.pone.0257454; @doi:10.1016/j.coesh.2020.09.003].
+To date, studies show that wastewater surveillance is an effective tool to monitor SARS-CoV-2 spread over large sewersheds [@doi:10.1016/j.scitotenv.2020.138875; @doi:10.1016/j.coesh.2020.09.006; @doi:10.1016/j.coesh.2020.09.003; @doi:10.1039/D1EW00747E]. 
+Indeed, data from a study in New York City indicated that wastewater SARS-CoV-2 detection correlated with clinical detection of infection  [@doi:10.1039/D1EW00747E].
+Similar studies have been conducted in Nevada [@doi:10.1016/j.wroa.2020.100086] and Boston [@doi:10.1128/mSystems.00614-20].
+To date, studies have shown that factors such as temperature, the travel time of wastewater, and diurnal variability may affect detection of SARS-CoV-2 [@doi:10.1016/j.scitotenv.2020.138875; @doi:10.1016/j.wroa.2020.100086].
```
Comment (rando2) [inline content/09.evolution.md:395]:
```suggestion
To date, studies have shown that factors such as temperature, the travel time of wastewater, and diurnal variability may affect detection of SARS-CoV-2 [@doi:10.1016/j.scitotenv.2020.138875; @doi:10.1016/j.wroa.2020.100086].
Additionaly, wastewater surveillance provides a tool to monitor fluctuations in the viral strains present in a community [@doi:10.1101/2022.02.18.22271188 ; @doi:10.1101/2021.12.21.21268077].
```

Comment (rando2) [reply content/09.evolution.md:395]: @RLordan those look good! Maybe something like this?

Comment (RLordan) [reply content/09.evolution.md:395]: > @RLordan those look good! Maybe something like this?

Yes, superb

**Pull request merged**